pyclient/setup.py

```diff
@@ -0,0 +1,20 @@
+from setuptools import setup
+
+setup(
+    name='deephaven',
```

Comment:
I wonder if we should have a distinct name for this package, so we don't confuse it for the package installed on the server?

Reply:
I agree that there is a problem.  With the same name, we have confusion between client and server, and we have problems with generating pydocs in a reasonable way.

Maybe `deephaven_client`?